Lets Do This/Controllers/Settings/LDTSettingsViewController.m

```diff
@@ -160,6 +170,16 @@ - (void)handleRateTap:(UITapGestureRecognizer *)recognizer {
     [[UIApplication sharedApplication] openURL:[NSURL URLWithString:@"itms-apps://itunes.apple.com/app/998995766"]];
 }
 
+- (void)handleTermsTap:(UITapGestureRecognizer *)recognizer {
+    [[GAI sharedInstance] trackEventWithCategory:@"behavior" action:@"tap on terms of service" label:nil value:nil];
```

Comment:
@jonuy -- added this new behavior to [wiki](https://github.com/DoSomething/product/wiki/Mobile-App#user-behavior)